twoslab.md

```diff
@@ -0,0 +1,42 @@
+# Two Slab Model
+![viewshieldblock701](viewshieldblock701.png)<br/></br>
+The simple two slab model geometry consists of two slabs and a cube that holds an isotropic 14 MeV source.
+#### Goals of Example

```

Comment:
This different heading levels have semantic meaning that we're kind of abusing...  should we just use bold here?

Reply:
Yes, I got a bit sloppy with md syntax copying from a different document.